components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -4,70 +4,19 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-import { useEffect, useState } from "react";
+import { useContext } from "react";
 import Modal from "../components/Modal";
-import TabMenuItem from "../components/TabMenuItem";
+import RepositoryFinder from "../components/RepositoryFinder";
+import { StartWorkspaceModalContext } from "./start-workspace-modal-context";
 
-export interface WsStartEntry {
-    title: string
-    description: string
-    startUrl: string
-}
-
-interface StartWorkspaceModalProps {
-    visible: boolean;
-    recent: WsStartEntry[];
-    examples: WsStartEntry[];
-    selected?: Mode;
-    onClose: () => void;
-}
-
-type Mode = 'Recent' | 'Examples';
+export function StartWorkspaceModal() {
+    const { isStartWorkspaceModalVisible, setIsStartWorkspaceModalVisible } = useContext(StartWorkspaceModalContext);
 
-export function StartWorkspaceModal(p: StartWorkspaceModalProps) {
-    const computeSelection = () => p.selected || (p.recent.length > 0 ? 'Recent' : 'Examples');
-    const [selection, setSelection] = useState(computeSelection());
-    useEffect(() => { !p.visible && setSelection(computeSelection()) }, [p.visible, p.recent, p.selected]);
-
-    const list = (selection === 'Recent' ? p.recent : p.examples).map((e, i) =>
-        <a key={`item-${i}-${e.title}`} href={e.startUrl} className="rounded-xl group hover:bg-gray-100 dark:hover:bg-gray-800 flex p-4 my-1">
-            <div className="w-full">
-                <p className="text-base text-gray-800 dark:text-gray-200 font-semibold">{e.title}</p>
-                <p>{e.description}</p>
-            </div>
-        </a>);
-
-    return <Modal onClose={p.onClose} visible={p.visible}>
+    return <Modal onClose={() => setIsStartWorkspaceModalVisible(false)} onEnter={() => false} visible={!!isStartWorkspaceModalVisible}>
         <h3 className="pb-2">New Workspace</h3>
```

Comment:
**suggestion**: What do you think of keeping the Open in Gitpod term here? New Workspace seems also ok. Your call. Added below some exports from the specs. 🏓 

| BEFORE | AFTER |
|-|-|
| <img width="612" alt="Frame 381" src="https://user-images.githubusercontent.com/120486/151395449-cc8d666a-ab20-4e05-97b7-5d2a1a62a6b4.png"> | <img width="612" alt="Frame 382" src="https://user-images.githubusercontent.com/120486/151395453-73102ca7-a4a4-4766-889d-c714e0747c92.png"> |